src/screens/GenderSelect.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  Pressable,
+  TouchableOpacity,
+} from 'react-native';
+
+type RadioButtonProps = {
+  gender: string;
```

Comment:
Genderのenumを追加してこう書くと良さそう！
```suggestion
  gender:Gender;
```